x-pack/plugins/apm/server/lib/anomaly_detection/get_anomaly_timeseries.ts

```diff
@@ -58,18 +79,24 @@ export async function getAnomalyTimeseries({
       j.datafeedState !== undefined && j.environment === preferredEnvironment
   )?.bucketSpan;
 
-  // If the calculated bucketSize is smaller than the bucket span interval,
-  // use the original job's bucket_span
-  const minBucketSize = preferredBucketSpan
-    ? parseInterval(preferredBucketSpan)?.asSeconds()
-    : undefined;
+  const minBucketSize =
+    parseInterval(
+      preferredBucketSpan ?? `${FALLBACK_ML_BUCKET_SPAN}m`
+    )?.asSeconds() ?? FALLBACK_ML_BUCKET_SPAN * 60; // secs
+
+  // Expected bounds (aka ML model plots) are stored as points in time, in intervals of the predefined bucket_span,
+  // so to query bounds that include start and end time
+  // we need to append bucket size before and after the range
+  const rangeFrom = start - minBucketSize * 1000; // ms
+  const rangeTo = end + minBucketSize * 1000; // ms
```

Comment:
Let's keep the original naming of `start`/`end` but make it clear what the difference is:

```suggestion
  const extendedStart = start - minBucketSize * 1000; // ms
  const extendedEnd = end + minBucketSize * 1000; // ms
```

Reply:
Updated here https://github.com/elastic/kibana/pull/132975/commits/282162712bdef24162969087afb81465c17adab6